ansible/roles/software/ckan/inventory/defaults/main.yml

```diff
@@ -49,6 +49,7 @@ inventory_datastore_ro_user: ckan_datastore_ro
 inventory_datastore_user: ckan_datastore
 inventory_db_name: ckan
 inventory_db_user: ckan
+inventory_python_home: "{{ python_home }}"
```

Comment:
Role defaults are true defaults, so we should really set a value here (and not refer to a potentially undefined variable). I _think_ `/usr` is the right value here. This value is confusing, longstanding technical debt 😩